src/app/modules/shared_modules/add_account/controller.nim

```diff
@@ -132,6 +132,9 @@ proc getKeypairs*(self: Controller): seq[KeypairDto] =
 proc getKeypairByKeyUid*(self: Controller, keyUid: string): KeypairDto =
   return self.walletAccountService.getKeypairByKeyUid(keyUid)
 
+proc getIndexForNextAccountNameSuggestion*(self: Controller): int =
```

Comment:
do we really need to have solution from status go?
Can't it just be <number of accounts> +1 since it's only a placeholder?

Reply:
> do we really need to have solution from status go?

Guess yes, don't see other way to do it on client side.

> Can't it just be +1 since it's only a placeholder?

@J-Son89 to add 1 to what?

Reply:
the number of accounts the user currently has on that device.

Reply:
issues with syncing etc won't really matter then because it's only a placeholder, so we don't need to have a solution to catch these edge cases.

Reply:
The request says:
```
When the user subsequently creates a new account, that account should be named Account 2, with the number iteratively increasing by 1 for each new account thereafter.

If a user removes an account, this should not be taken into account in the generation of the placeholder name. I.e. if I create Account 8 and then remove it, and then create a new account, this new account should have a placeholder name of Account 9 not Account 8.
```
and that's not possible to maintain on the client side only, we need statusgo for that. Also that index won't be synced, it's always calculated based on keypairs (keypairs are the thing for syncing).

Reply:
@benjthayer, @xAlisher is it so important that the placeholder value is following this logic? ^ ^

If you keep it simple and set the place holder as  `"Account " + <number of accounts user has on device> +1` then sure sometimes the placeholder value is slightly incorrectly, however the collateral damage from that is so insignificant that we can then avoid adding this extra logic to Status-Go.

Reply:
or better yet, just have a static placeholder name "New Account" or something :)